release-notes/unreleased/IGAPP-324-ReplaceOnClickOutside.yml

```diff
@@ -0,0 +1,5 @@
+issue_key: IGAPP-324
```

Comment:
Imo not necessary, see https://tuerantuer.slack.com/archives/CDV9QF1GA/p1605109647259600